wizard/class-instant-articles-option-amp.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Facebook Instant Articles for WP.
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @package default
+ */
+
+require_once( dirname( __FILE__ ) . '/class-instant-articles-option.php' );
+require_once( dirname(dirname( __FILE__ )) . '/class-instant-articles-amp-markup.php' );
+
+use Facebook\InstantArticles\Validators\Type;
+
+/**
+ * AMP Generation Settings
+ */
+class Instant_Articles_Option_Amp extends Instant_Articles_Option {
+
+	const OPTION_KEY = 'instant-articles-option-amp';
+
+	public static $sections = array(
+		'title' => 'AMP Markup Generation',
+		'description' => '<p>Settings to control the IA to AMP conversion, check the <a href="https://github.com/facebook/facebook-instant-articles-sdk-extensions-in-php" target="_blank">project\'s repository</a> for more info.</p>',
+	);
+
+	public static $fields = array(
+
+		Instant_Articles_Amp_Markup::SETTING_AMP_MARKUP => array(
+			'label' => 'Enable Markup',
```

Comment:
Let's add "(experimental)" to the label, as we want people to know that this is not on the same maturity as the rest of the plugin.